documentation/modules/con-kafka-connect-authentication.adoc

```diff
@@ -5,26 +5,29 @@
 [id='con-kafka-connect-authentication{context}']
 = Authentication support in Kafka Connect
 
-Authentication is configured through the `authentication` property in `KafkaConnect.spec` and `KafkaConnectS2I.spec`.
-The `authentication` property specifies the type of the authentication mechanisms which should be used and additional configuration details depending on the mechanism.
-The supported authentication types are:
+You can configure authentication for Kafka Connect in the `authentication` property in `KafkaConnect.spec` and `KafkaConnectS2I.spec`. 
+The `authentication` property specifies which of the supported authentication mechanisms to use, and additional configuration depending on the mechanism.
+
+The following types of authentication are supported for Kafka Connect:
 
 * TLS client authentication
 * SASL-based authentication using the SCRAM-SHA-512 mechanism
 * SASL-based authentication using the PLAIN mechanism
 * xref:assembly-oauth-authentication_str[OAuth 2.0 token based authentication]
 
-== TLS Client Authentication
+The following sections explain how each authentication mechanism works. 
+
+== TLS Client authentication
```

Comment:
I'm not sure we need this section. I think we could just cross-ref to https://strimzi.io/docs/operators/master/using.html#type-KafkaClientAuthenticationTls-reference as we're moving descriptions of properties into the appendix.